code/datums/components/hallucinations.dm

```diff
@@ -0,0 +1,537 @@
+TYPEINFO(/datum/component/hallucination/trippy_colors)
```

Comment:
Shouldn't this be below the parent definition? if there's a reason to have it up here please comment that somewhere

Reply:
These are up here because it's easier to read the arguments for all the components at the top of the file, rather than having to find each one in there.